### ISSUE TITLE
Allow Consecutive GitHub Alerts

### ISSUE THREAD

**Issue by gabeseltzer** [DavidAnson/vscode-markdownlint#360]
GitHub uses the following syntax for ["Alerts"](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts):

```markdown
> [!NOTE]
> Useful information that users should know, even when skimming content.
```

If you put two alerts consecutively like this, you get a warning about: `MD028/no-blanks-blockquote: Blank line inside blockquote`

```markdown
> [!NOTE]
> my first note.

> [!NOTE]
> A second, consecutive note...
```

Can we change the parsing for the rule so that it recognizes the kinds of alerts and allows spaces between them?
